code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -48,9 +48,10 @@
 	set category = "Malfunction"
 	set name = "Upgrade Turrets"
 	src.verbs -= /mob/living/silicon/ai/proc/upgrade_turrets
-	for(var/obj/machinery/turret/turret in machines)
-		turret.health += 30
-		turret.shot_delay = 20
+	for(var/obj/machinery/porta_turret/turret in machines)
+		if(turret.ai) //Make sure only the AI's turrets are affected.

```

Comment:
Yes, I am aware of that, but it is the only practical way to differentiate between turrets built by players and the ones pre-placed on the map. If it is better to omit this check, thus affecting any and all existing portable turrets instead, I will do that.